buildozer/default.spec

```diff
@@ -221,6 +221,9 @@ fullscreen = 0
 # (str) The Android arch to build for, choices: armeabi-v7a, arm64-v8a, x86, x86_64
 android.arch = armeabi-v7a
 
+# (int) overrides automatic versionCode computation (used in build.gradle)
+# android.numeric_version = 1
```

Comment:
Actually could you add a comment noting that this is not the same as the app version, and should only be edited if you know what you're doing?

Reply:
Good point I've added `this is not the same as app version and should only be edited if you know what're doing`

Reply:
Typo "what're" -> "what you're" :p

Reply:
And this is why I love code review when it's possible
I've rushed it because I'm exited about the feature :)
Thanks addressed!

Reply:
Now it's "yo're"

Reply:
Yes I saw it as I pushed and already fixed :facepalm:

Reply:
I'm happy to push force as many time as needed :smile: